src/Sylius/Component/Shipping/Model/Shipment.php

```diff
@@ -22,6 +23,8 @@
  */
 class Shipment implements ShipmentInterface, TimestampableInterface
```

Comment:
The major thing is that we display `createdAt` field in the admin backend. When someone will implement it's own `Shipment` class and doesn't define `createdAt`, we will have an exception during page render.